### PR TITLE
Filter dashboard by topic

### LINKED ISSUE #10: Allow filtering by topic in eMQ dashboard
# Goal

In the current APInf dashboard, it is possible to select one API backend at a time for visualisation. We would like a similar feature for the eMQ dashboard, so a topic owner could see analytics for one topic.

# Definition of done
Topic owner can see analytics for a given topic. For example, subscriptions and published messages for one topic only.
(Details to be added by assignee.)

### PR DESCRIPTION
**Pull request by frenchbread**
Closes #10 #11 

### Changes
- Updated queries and fields to fit `emq-plugin-elasticsearch`
- Added `filterTopic` component for filtering data by emq topic
- Added meteor methods:
    - `getTopics`
    - `getEventTypes`
- Refactored `eventType-filter` to use dynamically fetched data, not hardcoded
- Various fixes and cleanups
- Updated README

### CODE REVIEW

**Comment by frenchbread** [comment]
Please review.

**Comment by brylie** [comment]
Please lint this code using the same ESLint rules as Apinf.

- https://github.com/apinf/platform/blob/develop/.eslintrc
- https://github.com/apinf/platform/blob/develop/package.json#L39-L45

File: dashboard/client/filter/topic/topic.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          value: t.key,
+          text: t.key
+        }))
+      })
+
+      // Update "select" is required to initialize bootstrap-select
+      // with fresh data
+      $('#filter-topic').selectpicker('refresh')
+
+      // Update UI selecttion with value (if one exists)
```
Comment (brylie) [inline dashboard/client/filter/topic/topic.js]: Typo

**Comment by frenchbread** [comment]
Done

**Pull request merged**